hack/conformance.sh

```diff
@@ -22,22 +34,35 @@ if [[ ! -z "$DOCKER_TAG" ]]; then
 fi
 
 if [[ ! -z "$KUBEVIRT_E2E_FOCUS" ]]; then
-    sonobuoy_args="${sonobuoy_args} --plugin-env kubevirt-conformance.E2E_FOCUS=${KUBEVIRT_E2E_FOCUS}"
+    add_to_label_filter "(${KUBEVIRT_E2E_FOCUS})" "&&"
```

Comment:
Can you explain in the commit message why label filters are better than focus/skip?

```
add storage related opt outs in conformance script
and refactor the script to use label filters instead of focus/skip
```

Reply:
done